src/github/utils/config.ts

```diff
@@ -0,0 +1,53 @@
+import { Value } from "@sinclair/typebox/value";
+import { GitHubContext } from "../github-context";
+import YAML from "yaml";
+import { Config, configSchema } from "../types/config";
+
+const UBIQUIBOT_CONFIG_FULL_PATH = ".github/ubiquibot-config.yml";
+
+export async function getConfig(context: GitHubContext): Promise<Config | null> {
+  const payload = context.payload;
+  if (!("repository" in payload) || !payload.repository) throw new Error("Repository is not defined");
```

Comment:
I'm not sure how we can make this less verbose

Reply:
You mean how the error renders in the console with the massive stack trace filled with `node_modules`?

Reply:
ChatGPT says 
```suggestion
if (!payload.repository) throw new Error("Repository is not defined");
```

is sufficient.

Reply:
I agree but I get this error:
```
 Property 'installation' does not exist on type 'RepositoryDispatchEvent | BranchProtectionRuleCreatedEvent | BranchProtectionRuleDeletedEvent | ... 417 more ... | WorkflowDispatchEvent'.
  Property 'installation' does not exist on type 'never'.
```